server/protocols/jmap/src/main/java/org/apache/james/jmap/model/VacationResponse.java

```diff
@@ -71,7 +71,7 @@ public Builder textBody(String textBody) {
         }
 
         public Builder fromVacation(Vacation vacation) {
-            this.id = vacation.getId();
+            this.id = Vacation.ID;
```

Comment:
Strange change. Maybe you should ensure in builder that id is set to "singleton"?